### ISSUE TITLE
better accessibility

### ISSUE THREAD

**Issue by codeMrRobot** [Loulou2018/Aloha-Apparel-Co._project1#2]
insufficient contrast ratio. The color combination #e2574c/#ffffff has a contrast ratio of 3.68, which is not sufficient. At this size, you will need a ratio of at least 4.5 and other ratio listed below
3.25<4.5
1<4.5 
1>3
